src/04-arrays.js

```diff
@@ -65,7 +77,10 @@ function getFirstElementOfArray(array) {}
  *  getLastElementOfArray([ null, undefined ]);
  *  //> undefined
  */
-function getLastElementOfArray(array) {}
+function getLastElementOfArray(array) {
+  
+  return array.pop(array.length-1);
```

Comment:
Take another look at what ` .pop() ` does.  What arguments does this function take?